docs/administration/vulnerability-scanning/_index.md

```diff
@@ -3,13 +3,25 @@ title: Vulnerability Scanning
 weight: 35
 ---
 
-Harbor provides static analysis of vulnerabilities in images through the open source [Clair](https://github.com/coreos/clair) project. 
+Harbor provides static analysis of vulnerabilities in images through the open source projects [Trivy](https://github.com/aquasecurity/trivy) and [Clair](https://github.com/coreos/clair). To be able to use Trivy, Clair or both you must have enabled Trivy, Clair or both when you installed your Harbor instance.
```

Comment:
it should be 'you must install Trivy or Clair during your installation of Harbor or install these through interrogation service (add link) and then activate them at a later time.

Reply:
About what you said, it's in the next paragraph that explicitly talks about how to use interrogation service. Here we only talk about the "default" scanner things.